### DISCUSSION TITLE
Oven Siemens

### DISCUSSION THREAD

**Discussion by Dabbadoeber** [alexkn/node-red-contrib-homeconnect#29] [Q&A]
Hello i get 

object
_msgid: "2c2c293c.a5e0e6"
payload: 1612295426851
topic: ""
error: "Error: Conflict"

on everything i do. 
I want to get some status information of my oven, but i do not get it.
Get_apliances is working
Get_Specific_apliances is working 
but this is about it. 

![autherzization](https://user-images.githubusercontent.com/9031962/106655898-41e7a080-659a-11eb-9efc-c04c72a8ad1b.png)

![Get_status](https://user-images.githubusercontent.com/9031962/106655912-4613be00-659a-11eb-909f-e335391f3089.png)

i have the HAID from the Oven, on the scope tab in my account oven is checked,  accept for control.

Gr. Rick

**Comment by alexkn** [comment]
Maybe you are missing the "Additional Partner Agreement".
https://developer.home-connect.com/docs/authorization/scope
I don't have a device which needs that.

**Reply by leifnel** [reply]
I have an oven that requires this, but it's not clear for me to get this.

**Comment by leifnel** [comment]
Oven control is not available for us mere mortals yet. See https://github.com/alexkn/node-red-contrib-homeconnect/discussions/35
It should be available summer 2021

**Comment by alexkn** [comment]
According to the last Home Connect Developer newsletter, Oven Control should now be available for all clients. 
@Dabbadoeber @leifnel 
Have you tried this yet?
